File: ProjetUfFinal/Assets/BDD/CRUD LEVEL/levelInsert.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class levelInsert : MonoBehaviour
{
    string URL = "http://localhost/mydb/levelInsert.php";
    public string InputNom, InputCreateur, InputCreation, InputResultat;
    
    // Start is called before the first frame update
    void Start()
    {
        
    }

    // Update is called once per frame
    void Update()
    {
        if(Input.GetKeyDown(KeyCode.Space)){
            AddLevel (InputNom,InputCreateur,InputCreation,InputResultat);
        }
    }

    public void AddLevel(string nom, string createur, string creation, string resultat){
        WWWForm form = new WWWForm ();
        form.AddField ("addNom", nom);
        form.AddField ("addCreateur", createur);
        form.AddField ("addCreation", creation);
        form.AddField ("addResultat", resultat);

        WWW www = new WWW (URL, form);
    }
}

File: ProjetUfFinal/Assets/BDD/Fichiers_A_Mettre_Dans_Wamp/levelInsert.php
<?php
include('connexion.php');

$nom = $_POST['addNom'];
$createur = $_POST['addCreateur'];
$creation = $_POST['addCreation'];
$resultat = $_POST['addResultat'];

$sql = "insert into level(nom, createur, creation, resultat) values ('".$nom."','".$createur."','".$creation."','".$resultat."')";
$result = mysqli_query($connect, $sql);
?>